docs/integrations/README.md

```diff
@@ -116,6 +116,7 @@ Airbyte uses a grading system for connectors to help users understand what to ex
 |[Oracle](./destinations/oracle.md)| Alpha |
 |[Postgres](./destinations/postgres.md)| Certified |
 |[Redshift](./destinations/redshift.md)| Certified |
-|[S3](./destinations/s3.md)| Certified |
+|[S3](./destinations/s3.md)| Alpha |
+|[AzureBlobStorage](./destinations/azureblobstorage.md)| Certified |
```

Comment:
Please put it in alphabetical order;
Maybe here it should be indicated as Alpha?

Reply:
done, thanks!